angularjs/angular.d.ts

```diff
@@ -708,7 +708,6 @@ declare module angular {
     // see http://docs.angularjs.org/api/ng.$window
     ///////////////////////////////////////////////////////////////////////////
     interface IWindowService extends Window {
-        [key: string]: any;
```

Comment:
> If someone wants to override this, he/she can merge with the Window interface and extend it ... My final argument is alignment with lib.d.ts.

Agree with you on both accounts. But this feature was added explicitly by user demand, and rather not break their `noImplicitAny` code bases needelessly. That :ship: has sailed. Please revert for `window` and `browser`